sklearn/metrics/pairwise.py

```diff
@@ -231,7 +231,11 @@ def pairwise_distances_argmin_min(X, Y, axis=1, metric="euclidean",
 
     Parameters
     ----------
-    X, Y : {array-like, sparse matrix}
+    X : {array-like, sparse matrix}
+        Arrays containing points. Respective shapes (n_samples1, n_features)
```

Comment:
Not arrays. I prefer the `X, Y` notation. Again, this is why I did not design the below script as a test, but as a way to list likely issues.